.vitepress/config.mts

```diff
@@ -89,132 +91,308 @@ export default defineConfig({
   themeConfig: {
     siteTitle: "OXC",
     logo: "https://raw.githubusercontent.com/oxc-project/oxc-assets/main/logo-round.png",
-    logoLink: "/",
```

Comment:
Without this option, VitePress will generate link for the current locale's top page.